### PR TITLE
Update: taking Share Kotlin Notebook out of get started.

### PR DESCRIPTION
**Pull request by AlejandraPedroza**
This PR contains updates to the Data analysis section to test improvement in user navigation according to GA analysis.
[KT-69577](https://youtrack.jetbrains.com/issue/KT-69577) Update Kotlin for data analysis docs

### CODE REVIEW

**Review by zaleslaw — APPROVED** [review]

File: docs/topics/data-analysis/data-analysis-notebooks-output-formats.md [hunk 1]
```diff
@@ -288,6 +288,7 @@ You can create various charts directly in your Kotlin Notebook to visualize your
...
 ## What's next
 
-* [Visualize data using the DataFrame and Kandy libraries](data-analysis-visualization.md)
-* [Retrieve data from the CSV and JSON files](data-analysis-work-with-data-sources.md)
-* [Check out the list of recommended libraries](data-analysis-libraries.md)
\ No newline at end of file
+* [Visualize data using the DataFrame and Kandy libraries](data-analysis-visualization.md).
+* [Learn more about rendering and displaying rich output in Kotlin Notebook](https://www.jetbrains.com/help/idea/kotlin-notebook.html#render-rich-output).
+* [Retrieve data from the CSV and JSON files](data-analysis-work-with-data-sources.md).
+* [Check out the list of recommended libraries](data-analysis-libraries.md).
```
Comment (daniCsorbaJB) [inline docs/topics/data-analysis/data-analysis-notebooks-output-formats.md:294]: Do we need periods at the end here as these are links? 🤔

File: docs/topics/data-analysis/kotlin-notebook-add-dependencies.md [hunk 2]
```diff
@@ -109,11 +108,8 @@ The resulting chart:
...
+## What's next
 
-## Next step
-
-In the next part of the tutorial, you will learn how to share a Kotlin Notebook.
-
-**[Proceed to the next chapter](kotlin-notebook-share.md)**
+* See more details about [adding dependencies to your Kotlin Notebook](https://www.jetbrains.com/help/idea/kotlin-notebook.html#add-dependencies).
+* For a more extensive guide using the Kotlin DataFrame library, see [Retrieve data from files](data-analysis-work-with-data-sources.md).
+* For an extensive overview of tools and resources available for data science and analysis in Kotlin, see [Kotlin and Java libraries for data analysis](data-analysis-libraries.md).
```
Comment (daniCsorbaJB) [inline docs/topics/data-analysis/kotlin-notebook-add-dependencies.md:115]: Could we add a link to the Share notebooks here?

**Review by daniCsorbaJB — COMMENTED** [review]
LGTM 🚀 - I  some minor comments

**Review by daniCsorbaJB — APPROVED** [review]
Thank you for the changes LGTM 🚀

**Pull request merged**